lang/Makefile

```
@@ -8,7 +8,7 @@ PHP_SOURCES = $(shell find ../ -maxdepth 1 -name \*.php) \
 all : extract mo
 
 twig :
-	../bin/twig-cache
+	pushd ../../../../ && bin/console galette:twig-cache $(shell basename $(abspath $(CURDIR)/..))
```

```suggestion
	cd ../../../../ && bin/console galette:twig-cache $(shell basename $(abspath $(CURDIR)/..)) && cd ${INSTALLDIR}
```

There was a mistake in my inital comment on core [PR#615](https://github.com/galette/galette/pull/615#issuecomment-2697020970). I edited it afterwards. Here is the rule that is actually working on my side.